src/components/auth/login.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { logUserIn } from '../../redux/users/userSlice';
+import AuthForm from './authform';
+
+const Login = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const userState = useSelector((state) => state.user);
+
+  const setToken = (token) => {
+    localStorage.setItem('token', token);
+  };
+
+  const setCurrentUser = (user) => {
+    const userObj = JSON.stringify(user);
+    localStorage.setItem('user', userObj);
+  };
+
+  const endpoint = 'https://sheltered-tor-84017.herokuapp.com/api/v1/auth/login';
+  const onLogin = (res) => {
+    setToken(res.data.jwt);
+    setCurrentUser(res.data.user);
+    dispatch(logUserIn(res.data.user));
+    navigate('/');
+  };
```

Comment:
Also, it'd be nice to have a notification when the user is logged in.
The backend already returns a response, you should go ahead and display that. There's no need to create your own message since this will make your work faster